numpy/ma/core.py

```diff
@@ -7495,8 +7496,7 @@ def allclose(a, b, masked_equal=True, rtol=1e-5, atol=1e-8):
         return False
     # No infs at all
     if not np.any(xinf):
-        d = filled(umath.less_equal(umath.absolute(x - y),
-                                    atol + rtol * umath.absolute(y)),
+        d = filled(less_equal(absolute(x - y), atol + rtol * absolute(y)),
```

Comment:
Does this actually change much? Calling the masked array equivalent might be a bit cleaner I can imagine then jumping the hoops of the ufunc machinery, so you just prefer it or is there more to this?